include/h2o.h

```diff
@@ -2203,11 +2215,21 @@ inline h2o_conn_t *h2o_create_connection(size_t sz, h2o_context_t *ctx, h2o_host
 #else
     conn->id = __sync_add_and_fetch(&h2o_connection_id, 1);
 #endif
+    conn->_uuid.is_initialized = 0;
     conn->callbacks = callbacks;
```

Comment:
```suggestion
    conn->callbacks = callbacks;
    conn->_uuid.is_initialized = 0;
```

Let's do things in the order they are defined.